soc/arm/st_stm32/stm32wb/power.c

```diff
@@ -26,12 +41,31 @@ void pm_power_state_set(struct pm_state_info info)
 		return;
 	}
 
+	/* Implementation of STM32 AN5289, proposed in STM32WB Cube Application
+	 * BLE_HeartRate
+	 * https://github.com/STMicroelectronics/STM32CubeWB/tree/master/Projects/P-NUCLEO-WB55.Nucleo/Applications/BLE/BLE_HeartRate
```

Comment:
If that's implementation of STM32 AN5289, is there a need to give Cube sample reference ?

Reply:
I prefer to keep both reference: AN5289 is the theoretical one, Cube example is the practical one (from where I copy/past :smiley:  )

Reply:
Cube example reference removed from code.
And added to PR description.